### PR TITLE
Add GH+GRMHD TCI for DG and FD grids

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

The TCIs are the same as for the GRMHD solver.

Only the `Add TciOnDg(Fd)Grid...` commits are new

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: src/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/TciOnFdGrid.hpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/TciOnFdGrid.hpp"
+
+namespace grmhd::GhValenciaDivClean::subcell {
+/*!
+ * \brief The troubled-cell indicator run on the DG grid to check if the
```
Comment (yoonso0-0) [inline src/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/TciOnFdGrid.hpp]: `(...) run on the FD grid to check if the corresponding DG solution is admissible.` (from existing TciOnFdGrid.hpp docs)

**Comment by nilsdeppe** [comment]
Rebased and squashed in the dox fix. Thanks for the quick review :)

**Review by kidder — APPROVED** [review]

**Comment by nilsdeppe** [comment]
@yoonso0-0 can you approve or request changes on this? Thanks!

**Review by yoonso0-0 — APPROVED** [review]

**Comment by yoonso0-0** [comment]
Yep :)

**Comment by nilsdeppe** [comment]
Test failure is from TensorExpression test using too many resources, which was fixed in #3616

**Pull request merged**